recipes/tasklogger/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "tasklogger" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 50a3a20c9d675dc0e40f8a7d532bf845526ffa98a21ed0cd5ed957fcd40e54c1
+
+build:
+  number: 0
```

Comment:
Can this be `noarch: python`?

https://conda-forge.org/docs/meta.html#building-noarch-packages